lib/MusicBrainz/Server/Report/ArtistsDisambiguationSameName.pm

```diff
@@ -0,0 +1,39 @@
+package MusicBrainz::Server::Report::ArtistsDisambiguationSameName;
+use Moose;
+
+with 'MusicBrainz::Server::Report::ArtistReport',
+     'MusicBrainz::Server::Report::FilterForEditor::ArtistID';
+
+sub query {
+    "
+        SELECT
+            artist.id AS artist_id,
+            row_number() OVER (ORDER BY musicbrainz_collate(artist.name), artist.id)
```

Comment:
It gives us consistent sorting using UCA. https://github.com/metabrainz/postgresql-musicbrainz-collate/blob/master/README.musicbrainz_collate.md gives a brief explanation.

I believe the builtin text sorting postgres uses with, say, `ORDER BY artist.name` depends on the lc_collate setting of the database, i.e. it's locale-specific. I'm not an expert on this, though.